evalml/automl/automl_algorithm/default_algorithm.py

```diff
@@ -70,6 +70,7 @@ class DefaultAlgorithm(AutoMLAlgorithm):
         num_long_pipelines_per_batch (int): number of pipelines per batch for each top n pipeline through long mode.
         allow_long_running_models (bool): Whether or not to allow longer-running models for large multiclass problems. If False and no pipelines, component graphs, or model families are provided,
             AutoMLSearch will not use Elastic Net or XGBoost when there are more than 75 multiclass targets and will not use CatBoost when there are more than 150 multiclass targets. Defaults to False.
+        features (list)[FeatureBase]: List of features to run DFS on in AutoML pipelines. Defaults to None. Features will only be computed if the columns used by the feature exist in the input and if the feature itself is not in input.
```

Comment:
I'm trying to convey that if `ABSOLUTE(1)` was passed in as a feature, it will only be computed if:
1. `1` is a column in the input
2. `ABSOLUTE(1)` doesn't already exist in the input

Will def try to reword though!